scripts/ripe-atlas

```diff
@@ -1,94 +1,10 @@
 #!/usr/bin/env python
-
-import importlib
-import os
-import pkgutil
-import re
 import sys
 
-from ripe.atlas.tools import commands
+from ripe.atlas.tools.script import RipeAtlas
```

Comment:
I would prefer also to have it in here unless there is another reason windows forces us to move it.